packages/core/package.json

```diff
@@ -31,6 +31,7 @@
     "prepack": "yarn build"
   },
   "dependencies": {
+    "@thi.ng/vectors": "^6.0.0",
```

Comment:
Can you say a bit more about why this dependency is needed?

Reply:
I added it because I originally thought I was going to use it, but it turns out its API was so annoying that it was easier to port my own vector ops (you can't write `add(v1, v2)`, you have to write `add(result, v1, v2)`). What do you think, do we want to keep a real vector math library around in the frontend or just roll our own? Caveat, whatever I'm doing is probably not amazing for complicated numerics but it's just fine for the current use case.

Reply:
I see. These impure APIs can be annoying and error prone. I was mainly being careful about adding dependencies. If there are only one or two functions to write, I'm more inclined to (a) find a better library and use it widely or (b) drop this library and write your own functions